prinz/src/main/scala/pl/touk/nussknacker/prinz/mlflow/MlflowConstants.scala

```diff
@@ -1,8 +1,19 @@
 package pl.touk.nussknacker.prinz.mlflow
 
+import java.net.URL
+
 object MlflowConstants {
-  val DEFAULT_USER: String = "mlflow.user"
+  val DEFAULT_HOST: String = "http://localhost"
   val DEFAULT_PORT: Int = 5000
+  val DEFAULT_SERVE_PORT: Int = 1234
+  val DEFAULT_URL: URL = new URL(s"$DEFAULT_HOST:$DEFAULT_PORT")
+  val DEFAULT_USER: String = "mlflow.user"
   val BASE_API_PATH: String = s"/api/2.0/mlflow"
   val BASE_PREVIEW_API_PATH: String = s"/api/2.0/preview/mlflow"
+  val S3_ACCESS_KEY: String = "mlflow-key"
+  val S3_SECRET_KEY: String = "mlflow-secret"
+  val S3_URL: URL = new URL("http://localhost:9000")
```

Comment:
Let's use DEFAULT_HOST and define a constant for S3 port. We might be using S3 again later e.g. as PMML model registry.

Reply:
Will fix tonight, thanks